gradio/cli/commands/reload.py

```diff
@@ -56,16 +54,10 @@ def _setup_config(
         relpath = original_path
     module_name = str(relpath.parent / relpath.stem).replace(os.path.sep, ".")
 
-    gradio_folder = Path(inspect.getfile(gradio)).parent
```

Comment:
I removed the listener to the gradio folder itself, because there are many parts of the gradio the library that are not compatible with reloading - better to just remove the whole directory.

Reply:
Wait why are we dropping this? Now the reloader doesn't apply if you are changing gradio source files? I would regularly use this (and it would work fine).

Reply:
ok reverted